### PR TITLE
node name print bug fix with ros2 action info.

### PR DESCRIPTION
**Pull request by fujitatomoya**
closes https://github.com/ros2/ros2cli/issues/911

### CODE REVIEW

File: ros2action/test/fixtures/fibonacci_action_server.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 class FibonacciActionServer(Node):
 
     def __init__(self):
-        super().__init__('fibonacci_action_server')
+        super().__init__('fibonacci_action_server', namespace='/test')
```
Comment (fujitatomoya) [inline ros2action/test/fixtures/fibonacci_action_server.py:26]: adding the namespace here, so that we can check this as unit test.

**Comment by fujitatomoya** [comment]
@clalancette @sloretz can you take a look? i think we need to backport this to all distros.

**Review by christophebedard — APPROVED** [review]
Looks good to me with green CI.

**Review by clalancette — APPROVED** [review]
Good catch.  This looks good to me with green CI.

**Comment by fujitatomoya** [comment]
Pulls: ros2/ros2cli#926
Gist: https://gist.githubusercontent.com/fujitatomoya/a8582db356860824e900928997784c8b/raw/7f0117419eaa5df5f793b7e2e97bfb9df02232fe/ros2.repos
BUILD args:  --packages-up-to ros2action
TEST args:  --packages-select ros2action
ROS Distro: rolling
Job: ci_launcher
ci_launcher ran: https://ci.ros2.org/job/ci_launcher/14510
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/21785/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/16057/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/1288/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22484/)

**Review by ahcorde — APPROVED** [review]

**Pull request merged**